vncviewer/OptionsDialog.h

```diff
@@ -63,6 +63,7 @@ class OptionsDialog : public Fl_Window {
 
   static void handleX509(Fl_Widget *widget, void *data);
   static void handleRSAAES(Fl_Widget *widget, void *data);
+  static void handleViaHost(Fl_Widget *widget, void *data);
```

Comment:
This doesn't seem to be used?

Reply:
Thanks for catching this.  It was there in the original version before your new options work and when I reworked the code I accidentally left this in.   I'll remove it.